app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -60,6 +60,12 @@ import { addQuickAccessProfileItem } from "discourse/widgets/quick-access-profil
 import KeyboardShortcuts from "discourse/lib/keyboard-shortcuts";
 import { addFeaturedLinkMetaDecorator } from "discourse/lib/render-topic-featured-link";
 import { getOwner } from "discourse-common/lib/get-owner";
+import {
+  addInOptionsForUsers,
+  addInOptionsForAll,
+  addStatusOptions,
+  addPostTimeOptions,
+} from "discourse/components/search-advanced-options";
 
 // If you add any methods to the API ensure you bump up this number
 const PLUGIN_API_VERSION = "0.10.2";
```

Comment:
Please can you bump this number to 0.11.0, so that plugins can use that to detect whether the core version supports these new methods. (Then make sure to use that new number in discourse-assign, so the plugin keeps working on Discourse stable)